README.md

```diff
@@ -39,43 +40,48 @@ The below table lists various files used in correcting the bani portion of the d
 
 If one would like to proofread the database, they may click on the corresponding link for a PDF. The compare the proof against the source (Shabad OS Database) using the online viewer.
 
-### Bhai Nand Lal Ji
+### Sri Guru Granth Sahib Ji
 
-| Proof | Edition | Author | Publisher |
-| ----- | ------- | ------ | --------- |
-| Bhai Nand Lal Granthavali | 2009 (4th ed) | Dr. Ganda Singh | Punjabi University, Patiala |
-| Kalaam-e-Goya | 2003 (1st ed) | Pritpal Singh Bindra  | Institute of Sikh Studies, Chandigarh |
-| Rehitname | 2006 (8th ed) | Piara Singh Padam | Singh Brothers, Amritsar |
+Proof | Edition | Author | Publisher
+---|---|---|---
+[Shabadaarth (Vol. 1-4)](https://drive.google.com/open?id=1EF1MUYLPdJ_Qu2KZbp1g1lWvL1_IL2K8) | Vol. 1: October 2009<br> Vol. 2: August 2012<br> Vol. 3: July 2010<br> Vol. 4: September 2010 | - | SGPC, Sri Amritsar
+[Sri Guru Granth Sahib (Unicode)](https://drive.google.com/open?id=1_ozxV6_rmZ8kG1AQ9iZx2s49ZtnlU82E) | - | - | SGPC.net
 
-### Kabit Svaiye Bhai Gurdaas Ji
+### Sri Dasam Granth
 
-| Proof | Edition | Author | Publisher |
-| ----- | ------- | ------ | --------- |
-| Kabitt Swayye Bhai Gurdas Ji | 2007 (1st ed) | Shamsher Singh Puri | Singh Brothers, Amritsar |
-| Kabitt Sawaiye Bhai Gurdas Ji Steek | 2011 (7th ed) | Bhai Sewa Singh | Singh Brothers, Amritsar | 
+Proof | Edition | Author | Publisher
+---|---|---|---
+[Das Granthi](https://drive.google.com/open?id=16KaLDgZUNzTwghdYatRL3dGq9vnXLRP6) | March 2006 | - | SGPC, Sri Amritsar
+[Nitnem Gutka](https://drive.google.com/open?id=1AyQLCQgrNznDxI9Vby3knE4Piyrdsnxs) | May 2017 | - | SGPC, Sri Amritsar
+[Akal Ustat](https://drive.google.com/open?id=1QumuyIUC1oLRoODz3gaaEEnYwbqj1d5t) | 5th Ed., December 1995 | - | SGPC, Sri Amritsar
+[Bachittar Natak](https://drive.google.com/open?id=1z3Waum_FUoUzb1Faj7l5ZGzJYHG2THe1) | 7th Ed., July 2000 | - | SGPC, Sri Amritsar
+[Sri Dasam Granth](https://drive.google.com/open?id=1EG6okisvuibNK-RgSlLBLS1Z_T5-81_p) | - | Dr. Ratan Singh Jaggi | Gobind Sadan
```

Comment:
Does `Gobind Sadan` have a location? Is this a publisher or a person?

Reply:
Yup. http://www.singhbrothers.com/en/shri-dasam-granth-sahib-5-vol

Reply:
Please add `, New Delhi` to match the publisher pattern in other lines of the table